modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -145,6 +145,21 @@ spec:
 <10> Optional: Replace `<root_device_hint>` with a device path if you specify a root device hint.
 +
 <11> Optional: If you have configured the network interface for the newly created node, provide the network configuration secret name in the `preprovisioningNetworkDataName` of the BareMetalHost CR.
++
+<12> Optional: If you have configured the network interface with `nmstate`, and you want to ignore the DHCP configuration, you must disable the network interface. To disable the network interface, set the IP addresses to `enabled: false` as shown:
```

Comment:
I might re-word this a bit so instead of "you want to ignore the DHCP configuration" it says something like "you want to disable an interface". When people are running into this problem it's because they want to disable an interface and they may not necessarily equate ignoring DHCP to disabling it, even though in this case they're the same thing.

Also maybe mention the fact that state must be up? I know it's in the example, but that's the kind of unintuitive part here and it wouldn't hurt to reinforce it in the comment.

Reply:
I agree with the above re-wording. When I got myself into this situation it was because I wanted to disable an interface. I don't fully grok the NetworkManager interface up/down flow so I defer to @cybertron for that.

Reply:
Thank you for your comments, @cybertron and @mrbojangles3 
I have revised the description. PTAL and let me know if for any further changes.